src/lib/tabs/tabs.md

```diff
@@ -80,6 +80,31 @@ The `tab-nav-bar` is not tied to any particular router; it works with normal `<a
 the `active` property to determine which tab is currently active. The corresponding
 `<router-outlet>` can be placed anywhere in the view.
 
+## Lazy Loading
+By default, the tab contents are eagerly loaded. Eagerly loaded tabs
+will initalize the child components but not inject them into the DOM
+until the tab is activated. 
+
+If the tab contains several complex child components or the tab's contents
+rely on DOM calculations during initialization, it is advised
+to lazy load the tab's content. Tab contents can be lazy loaded by
```

Comment:
I think this statement should stand out a bit more, even just by making it a new paragraph:

"Tab contents can be lazy loaded by declaring the body in a `ng-template` with the `matTabContent` attribute."